control/tap-ctl-spawn.c

```diff
@@ -196,5 +232,15 @@ tap_ctl_spawn(void)
 	if (id < 0)
 		EPRINTF("get_id failed, child %d err %d\n", child, errno);
 
+	/* Put the tapdisk in a cgroup slice (best-effort) */
+	if (!slice) {
+#ifdef TAP_CTL_USE_DEFAULT_CGROUP_SLICE
```

Comment:
OK, now I know what you meant. I am OK to have it a compile time option (but in a near future
I'd rather have it a run-time one) but in this case the user needs to modify the source code, while
a middle ground approach would be to let people decide what to do while compiling.
So, what about this:
* call the macro "TAP_CTL_NO_CGROUP"
* remove the define directive
* change the ifdef to
```
#ifndef TAP_CTL_NO_CGROUP
```
Doing this, the default behaviour is the same as you pictured while giving the user the option to
pass -DTAP_CTL_NO_CGROUP and disable it.